pmd/rulesets/quickstart.xml

```diff
@@ -225,7 +225,7 @@
   >
       <priority>3</priority>
       <properties>
-        <property name="skipTestMethodUnderscores" value="true" />
+         <property name="testPattern" value="[a-z][a-zA-Z0-9_]*" />
```

Comment:
This regex looks like it applies to all filenames but shouldn't the regex only be specifying any filename that includes the word Test?

Reply:
testPattern is for test methods only, you can check this for more https://docs.pmd-code.org/pmd-doc-7.0.0/pmd_rules_apex_codestyle.html#methodnamingconventions